.gitignore

```diff
@@ -0,0 +1,12 @@
+.DS_Store
```

Comment:
This should be in your global system wide gitignore ;)

Reply:
👍 It was in the default `.gitignore` file of docusaurus